priv/emqx.schema

```diff
@@ -1715,6 +1725,16 @@ end}.
   {datatype, string}
 ]}.
 
+{mapping, "listener.wss.$name.proxy_address_header", "emqx.listeners", [
+  {default, "X-Forwarded-For"},
```

Comment:
The default value here means that the behavior of parsing `X-Forward-*` is turned on by default, is this purposely done here?

My opinion is that, first of all, this is different from the previous design (in 2.3.11). The previous logic required the configuration to be manually enabled to allow parsing of `X-Forward-*`; when the configuration was commented out, it meant that `X-Forward-*` would not be parsed.

Reply:
By default we should turn this off, is this ok? 

Or at least it should allow the user a way to turn it off.